pkg/shell/shell.go

```diff
@@ -56,6 +69,11 @@ func RunShellLine(config ShellConfig, line string) error {
 		return err
 	}
 
+	go func() {
+		<-signals
+		shellInstance.CancelQuery()
```

Comment:
is there a chance that the cancellation fails or at least take more time than expect? maybe in that case we should have something to communicate with the user